airbyte-config/config-models/src/main/resources/types/ReplicationAttemptSummary.yaml

```diff
@@ -11,6 +11,12 @@ required:
   - endTime
   - totalStats
   - streamStats
+  - replicationStartTime
```

Comment:
we may want to put these fields on the totalStats object instead of here. I think the existing stats are actually deprecated since we're using the data from totalStats and streamStats instead

Reply:
I think that's better! I was matching the position of `startTime` and `endTime`, which perhaps should be moved into totalStats one day as well?

Reply:
Updated via https://github.com/airbytehq/airbyte/pull/17643/commits/0c6f93ebcfb716b674f1f6b4e7ac290ccda80ba9

Reply:
Oh interesting, I missed startTime & endTime. are those different from these new replicationStart and end time stats?

Reply:
Yep - startTime and endTime would be T0 and T4 on the chart above. replicationStart is (more or less) T0, but replicationEnd is T3